Header header

string vehicleName
int32 priority

bool moving
bool planning
bool removing
bool inFailure

gridData grid

string[] WrongRobots

int32[] xStop
int32[] yStop

uint32 currPathSeq
uint32 currPathSeg
